### ISSUE TITLE
Not receiving Himwari-8 or Meteosat

### ISSUE THREAD

**Issue by Mopar44084** [pietern/goestools#143]
I am using my satellite on GOES 16. In the /home/pi/ directory I do not see Himwari-8 or Meteosat folder. 
I did enable EMWIN and I do see it in that folder. I have updated goestools.

**Comment by zeke800** [comment]
I think Himawari-8 is only broadcast on GOES-17. Check if it's in your visibility. The same thing happened to me and I was very disappointed until I found that out.

**Comment by Mopar44084** [comment]
I'm receiving Himawari-8 through GOES-16 EMWIN about every hour.

**Comment by JVital2013** [comment]
> I'm receiving Himawari-8 through GOES-16 EMWIN about every hour.

This is correct, but high resolution imagery is not available in its own folder. It's only available in low resolution EMWIN folder on GOES-16.

As far as I know, imagery that comes in on the EMWIN virtual channels cannot be split out into their own folders without secondary scripts
